components/common/IdentityForm.vue

```diff
@@ -101,10 +114,21 @@
       </p>
 
       <NeoButton
+        v-if="!hasIdentity"
         class="is-flex is-flex-grow-1 fixed-height"
         variant="k-accent"
         :label="$t('identity.create')"
-        :disabled="disabled"
+        :disabled="disabled || isAssetHub"
+        :loading="isLoading"
+        expanded
+        @click="openConfirmModal" />
+
+      <NeoButton
+        v-if="hasIdentity"
+        class="is-flex is-flex-grow-1 fixed-height"
+        variant="k-accent"
+        :label="$t('identity.update')"
+        :disabled="disabled || isAssetHub"
```

Comment:
doesn't need 2 button, you can only use one

Reply:
done